packages/web-components/src/horizontalbarchart/horizontalbarchart.stories.ts

```diff
@@ -13,6 +13,12 @@ export default {
 
 export const Basic: Story<FluentHorizontalBarChart> = renderComponent(storyTemplate).bind({});
 
+export const NMVariant: Story<FluentHorizontalBarChart> = renderComponent(html<StoryArgs<FluentHorizontalBarChart>>`
```

Comment:
lets move NM variant to a separate PR and merge this one.

Reply:
Okay sure